docs/user-guide/cli.md

```diff
@@ -64,6 +64,10 @@ Disable the default ignores. Stylelint will not automatically ignore the content
 
 Automatically fix, where possible, problems reported by rules. [More info](options.md#fix).
 
+### `--compute-edit-info, --cei`
+
+Compute edit information for fixable problems.
```

Comment:
[note] Not a blocker. We should add a link below once the documentation (`options.md`) is updated.

```suggestion
Compute edit information for fixable problems. [More info](options.md#computeeditinfo).
```